### ISSUE TITLE
Notes Stopped Syncing

### ISSUE THREAD

**Issue by UTexas80** [mozilla/notes#614]
Hello...My Notes have stopped syncing and no longer appear in my Sidebar. Is there anyway to restore my work?
Thank you.

**Comment by Natim** [comment]
Try to disconnect and reconnect?

**Comment by UTexas80** [comment]
Hi Natim - I appreciate your prompt response. Can you please help me understand, are you suggesting that I uninstall Test Pilot and re-install it or something else. I have several Firefox user accounts and Notes is displaying a blank screen for all of them so it doesn't appear to be account related.
Thank you.
Glen

**Comment by Natim** [comment]
So maybe you are reaching this error #512 ?

**Comment by Natim** [comment]
Could you provide me with a screenshot?

**Comment by UTexas80** [comment]
Yes, exactly - that's what's happening. I really love the Notes feature and rely on it heavily. I feel lost without it.

**Comment by Natim** [comment]
Which operating system are you using?

**Comment by Natim** [comment]
Can you help us get more information about why it stopped working on your end?

**Comment by UTexas80** [comment]
Surely. What information can I provide to you?

**Comment by Natim** [comment]
If you could run a debug enabled version of FIrefox and try to load notes in it and grab all the logs that you can about QuotaManager and things around it, it would help us to figure out why IndexedDB stopped working on your profile.

If you are using Windows, you can use this steps:

> For latest release build:
> 1. Go to https://treeherder.mozilla.org/#/jobs?repo=mozilla-release
> 2. Check the latest changeset (in the top)
> 3. Find "Windows 2012 debug" or "Windows 2012 x64 debug" row
> 4. Click on "tc (+3)"
> 5. Click on "B"
> 6. Find "artifact uploaded: target.zip" in the "Job Details" tab
> 7. Click on target.zip
> 8. Download target.zip
> 9. Unpack target.zip

Then can you open the `cmd` tool and drag and drop `firefox.exe` to run it and see the log in the console. We need the log talking about `QuotaManager`, `ActorsParent` or `kinto`.

**Comment by Natim** [comment]
@UTexas80 I am closing this bug as a duplicate of #512 please add you log there if you manage to get them 👍 thanks

**Comment by UTexas80** [comment]
Hi Natim - I got up to the "Then can you open the cmd tool" step but am unsure by which cmd tool? Do you mean the windows command prompt? BTW, I am able to read my notes using my home PC running Windows 7.

**Comment by Natim** [comment]
> Do you mean the windows command prompt?

Yes

> I am able to read my notes using my home PC running Windows 7.

Good news !!!

**Comment by UTexas80** [comment]
Got Firefox to start but was blocked by my work firewall to initiate it.

**Comment by Natim** [comment]
Ok :/

**Comment by UTexas80** [comment]
It looks like this issue for me is profile related. I used a different profile and notes worked. I went back to my original profile and notes was still hosed.

**Comment by Natim** [comment]
Yes I know, that's why if you can open a Firefox debug version with this profile we will be able to figure out why.

**Comment by UTexas80** [comment]
Hello. Here's my log from my Windows 7 x64 PC. Hope it helps

Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\gfalk>C:\Users\gfalk\Downloads\target\firefox\firefox.exe
[7688, Main Thread] WARNING: dependent window created without a parent: file z:/
build/build/src/toolkit/components/startup/nsAppStartup.cpp, line 674
++DOCSHELL 0000000009A4A800 == 1 [pid = 7688] [id = {8629ca8c-f01a-475e-9f71-0bb
2b09b4722}]
++DOMWINDOW == 1 (0000000009A4B000) [pid = 7688] [serial = 1] [outer = 000000000
0000000]
[7688, Main Thread] WARNING: 'NS_FAILED(rv)', file z:/build/build/src/widget/win
dows/nsLookAndFeel.cpp, line 846
++DOMWINDOW == 2 (0000000009A4C000) [pid = 7688] [serial = 2] [outer = 000000000
9A4B000]
[7688, Main Thread] WARNING: 'NS_FAILED(GetAccentColor(unused))', file z:/build/
build/src/widget/windows/nsLookAndFeel.cpp, line 472
[7688, Main Thread] WARNING: NS_ENSURE_SUCCESS(rv, NS_ERROR_UNEXPECTED) failed w
ith result 0x80004005: file z:/build/build/src/extensions/cookie/nsPermissionMan
ager.cpp, line 1012
JavaScript error: jar:file:///C:/Users/gfalk/Downloads/target/firefox/omni.ja!/c
omponents/XULStore.js, line 70: Error: Can't find profile directory.
[7688, Main Thread] WARNING: '!mLocalStore', file z:/build/build/src/dom/xul/XUL
Document.cpp, line 2041
Unable to read VR Path Registry from C:\Users\gfalk\AppData\Local\openvr\openvrp
aths.vrpath
[GPU 5928, Main Thread] WARNING: NS_ENSURE_TRUE(InitStaticMembers()) failed: fil
e z:/build/build/src/modules/libpref/Preferences.cpp, line 4746
[GPU 5928, Main Thread] WARNING: NS_ENSURE_TRUE(InitStaticMembers()) failed: fil
e z:\build\build\src\obj-firefox\dist\include\mozilla/Preferences.h, line 90
[GPU 5928, Main Thread] WARNING: NS_ENSURE_TRUE(InitStaticMembers()) failed: fil
e z:/build/build/src/modules/libpref/Preferences.cpp, line 4578
Unable to read VR Path Registry from C:\Users\gfalk\AppData\Local\openvr\openvrp
aths.vrpath
[7688, Main Thread] WARNING: Unsupported TextureClient backend type: 'aMoz2DBack
end != gfx::BackendType::SKIA && aMoz2DBackend != gfx::BackendType::DIRECT2D &&
aMoz2DBackend != gfx::BackendType::DIRECT2D1_1', file z:/build/build/src/gfx/lay
ers/client/TextureClient.cpp, line 1255
[7688, Main Thread] WARNING: Unsupported TextureClient backend type: 'aMoz2DBack
end != gfx::BackendType::SKIA && aMoz2DBackend != gfx::BackendType::DIRECT2D &&
aMoz2DBackend != gfx::BackendType::DIRECT2D1_1', file z:/build/build/src/gfx/lay
ers/client/TextureClient.cpp, line 1255
[7688, Main Thread] WARNING: Unsupported TextureClient backend type: 'aMoz2DBack
end != gfx::BackendType::SKIA && aMoz2DBackend != gfx::BackendType::DIRECT2D &&
aMoz2DBackend != gfx::BackendType::DIRECT2D1_1', file z:/build/build/src/gfx/lay
ers/client/TextureClient.cpp, line 1255
[7688, Main Thread] WARNING: Unsupported TextureClient backend type: 'aMoz2DBack
end != gfx::BackendType::SKIA && aMoz2DBackend != gfx::BackendType::DIRECT2D &&
aMoz2DBackend != gfx::BackendType::DIRECT2D1_1', file z:/build/build/src/gfx/lay
ers/client/TextureClient.cpp, line 1255
[7688, Main Thread] WARNING: NSS will be initialized without a profile directory
. Some things may not work as expected.: file z:/build/build/src/security/manage
r/ssl/nsNSSComponent.cpp, line 1762
[7688, Main Thread] WARNING: site security information will not be persisted: fi
le z:/build/build/src/security/manager/ssl/nsSiteSecurityService.cpp, line 554
[7688, Main Thread] WARNING: NS_ENSURE_SUCCESS(rv, rv) failed with result 0x8000
4005: file z:/build/build/src/netwerk/base/nsChannelClassifier.cpp, line 619

**Comment by Natim** [comment]
I guess you error is `Error: Can't find profile directory.` Can you run `firefox -P` to select a profile and make sure the profile directory still exists?

**Comment by UTexas80** [comment]
Right back to it...Hello...My Notes have stopped syncing and no longer appear in my Sidebar. Is there anyway to restore my work?
Thank you.

**Comment by Natim** [comment]
Can you try: https://firefox-storage-test.glitch.me/

**Comment by Natim** [comment]
In order to fix it you can remove all cache folder safely `FIREFOX_PROFILE_FOLDER/storage/{default,permanent,temporary}/*/cache`

**Comment by UTexas80** [comment]
Thank you for responding so quickly. I greatly appreciate your support. I deleted the local cache but it looks like the storage is totally broken...

Overview:
Storage is broken. This is your first visit or all storage was automatically cleared.
Specific Subsystem Statuses:

LocalStorage
    Good: Totally Working. (fullyOperational)
QuotaManager
    Good: Totally Working. (fullyOperational)
IndexedDB
    Bad: Our test logic is broken, please copy and paste the contents of 'Debug Info' below and anything in the devtools console and send to :asuth. (unexpectedBreakage)
Cache API
    Bad: Totally Broken. (fullyBroken)

Debug Info:


A mutation operation was attempted on a database that did not allow mutations.
The operation is insecure.

{
  "v": 1,
  "curVersion": 60,
  "prevVersion": 0,
  "ls": {},
  "qm": {
    "lastWorkedIn": 60
  },
  "idb": {
    "persistentCreatedIn": 0,
    "persistentLastOpenedIn": 0,
    "clearDetectedIn": 0
  },
  "cache": {
    "firstCacheCreatedIn": 0,
    "unpaddedOpaqueCreatedIn": 0,
    "paddedOpaqueCreatedIn": 0
  }
}

**Comment by Natim** [comment]
Do you mind adding this information as a comment of that bug? https://bugzilla.mozilla.org/show_bug.cgi?id=1425146

**Comment by UTexas80** [comment]
Got it.
